third_party/autograph/autograph/core/converter_testing.py

```diff
@@ -59,11 +59,11 @@ def is_inside_generated_code():
     del frame
 
 
-class TestingTranspiler(api.PyToTF):
+class TestingTranspiler(api.PyToLib):
   """Testing version that only applies given transformations."""
 
-  def __init__(self, converters, ag_overrides):
-    super(TestingTranspiler, self).__init__()
+  def __init__(self, converters, ag_overrides, hooks_impl=hooks.OperatorBase()):
```

Comment:
As usual, I'll complain about misused plural:
```suggestion
  def __init__(self, converters, ag_overrides, hook_impl=hooks.OperatorBase()):
```

Reply:
I changed it, still I prefer the plural name here, we have multiple hooks or operators to overload. 
I decided to use `operator overload` in the name rather than `hook impl` which looks like something internal.